index.html

```diff
@@ -0,0 +1,164 @@
+                    </p> </div>
+                </div>
+            </article>
+            <article>
+                <div class="card-display">
+                    <div class="program-image">
+                        <img class="icon-image" src="./images/networkicon (4).svg" alt="Workshop Icon">
+                        <p class="program-name">Forum</p>
+                        <p class="program-text">
+                            Have the chance to share your thoughts and opinions with experts in the field
+                        </p>
+                    </div> 
+                    </div>
+                </article>
+                <article>
+                <div class="card-display">
+                <div class="program-image">
+                    <img class="icon-image" src="./images/networkicon (5).svg" alt="Workshop Icon">
+                    <p class="program-name">Workshop</p>
+                    <p class="program-text">
+                        Workshops on the various creativity done by HennaHubUk academy
+                    </p>
+                </div> 
+                </div>
+            </article>
+            <article>
+                <div class="card-display">
+                <div class="program-image">
+                    <img class="icon-image" src= "./images/lectureicon (1).svg" alt="Networking icon">
+                    <p class="program-name">Networking</p>
+                    <p class="program-text">
+                        Get connected with fan around the world and make new friends even after the summit
+                    </p>
+                </div> 
+                </div> 
+            </article> 
+            </section>
+            <button> JOIN HENNAHUB SUMMIT 2023</button>
+        </div>
+        <a href="#">SEE THE WHOLE PROGRAM</a>
+    </div>
+</section>
+<!-- Main Program Card Section starts -->
+
+<!-- Featured Guest section starts -->
+    <section class="featured-guests" id="featured-guests">
+        <h2>Featured Speakers</h2>
+        <hr class="hr-line">
+        <div class="grid-cards"></div>
+        <div class="speaker-holder"></div>
+      </section>
+      <!-- Featured Guest section ends -->
+
+      <div class="more-info">
+        More <i class="fa-sharp fa-solid fa-chevron-down"></i>
+      </div>
+      <div class="less-info">
+        Less <i class="fa-sharp fa-solid fa-chevron-up"></i>
+      </div>
+
+      <!-- Sponsors and partners section starts -->
+      <div class="sponsors">
+        <p>Partner</p>
+        <hr>
+        <div class="spons">
+          <img src="./images/mozilla.PNG" alt="mozilla_logo">
+          <img src="./images/google.PNG" alt="google_logo">
+          <img src="./images/naver.PNG" alt="naver_logo">
+          <img src="./images/fancybraidslogo1.png" alt="fancy_logo">
+        </div>      
+      </div>
+<!-- Sponsors and partners section ends -->
```

Comment:
- [ ] As mentioned by the previous reviewer kindly put this part in the `section` tag for semantics reasons 😁